desktop/use-desktop/container.md

```diff
@@ -48,13 +41,31 @@ The **container view** displays **Logs**, **Inspect**, **Terminal**, **Files**,
     - Use the **Clear terminal** icon in the top right-hand corner to clear the logs terminal. 
     - Select and view external links that may be in your logs. 
 
-- Select **Inspect** to view low-level information about the container. You can see the local path, version number of the image, SHA-256, port mapping, and other details.
+### Inspect
+
+- Select **Inspect** to view low-level information about the container. It displays the local path, version number of the image, SHA-256, port mapping, and other details.
```

Comment:
Since this only has one bullet point, can maybe remove the list styling:
```suggestion
Select **Inspect** to view low-level information about the container. It displays the local path, version number of the image, SHA-256, port mapping, and other details.
```

Reply:
Oops, good catch